src/server/system_services/stats_aggregator.js

```diff
@@ -910,43 +910,45 @@ function partial_cycle_parse_prometheus_metrics(payload) {
         (percentage_of_unhealthy_buckets > 0.5 && 4) ||
         (percentage_of_unhealthy_buckets > 0.3 && 3) || 0;
 
-    prom_report.instance().set_providers_physical_logical(providers_stats);
-    prom_report.instance().set_cloud_types(cloud_pool_stats);
-    prom_report.instance().set_num_unhealthy_pools(unhealthy_pool_count);
-    prom_report.instance().set_num_pools(pool_count);
-    prom_report.instance().set_unhealthy_cloud_types(cloud_pool_stats);
-    prom_report.instance().set_system_info({ name, address });
-    prom_report.instance().set_system_links(links);
-    prom_report.instance().set_num_buckets(buckets_num);
-    prom_report.instance().set_bucket_status(buckets);
-    prom_report.instance().set_resource_status(resources);
-    prom_report.instance().set_num_objects(objects_in_buckets);
-    prom_report.instance().set_num_unhealthy_buckets(unhealthy_buckets);
-    prom_report.instance().set_health_status(health_status);
-    prom_report.instance().set_num_unhealthy_bucket_claims(unhealthy_bucket_claims);
-    prom_report.instance().set_num_buckets_claims(bucket_claims);
-    prom_report.instance().set_num_objects_buckets_claims(objects_in_bucket_claims);
-    prom_report.instance().set_system_capacity(capacity);
-    prom_report.instance().set_reduction_ratio(reduction_ratio);
-    prom_report.instance().set_object_savings_physical_size(physical_size);
-    prom_report.instance().set_object_savings_logical_size(logical_size);
-    prom_report.instance().set_bucket_class_capacity_usage(usage_by_bucket_class);
-    prom_report.instance().set_projects_capacity_usage(usage_by_project);
-    prom_report.instance().set_accounts_io_usage(accounts_stats);
-    prom_report.instance().set_accounts_num(accounts_num);
-    prom_report.instance().set_total_usage(total_usage);
-    // TODO: Currently mock data, update with the relevant values.
-    prom_report.instance().set_rebuild_progress(100);
-    prom_report.instance().set_rebuild_time(0);
+    const core_report = prom_reporting.get_report(config.NOOBAA_CORE_METRICS_REPORT);
```

Comment:
Same as the previous comment regarding multiple `prom_reporting.get_report`

Reply:
same answer